microshift_updating/microshift-about-updates.adoc

```diff
@@ -0,0 +1,46 @@
+:_content-type: ASSEMBLY
+[id="microshift-about-updates"]
+=  About {product-title} updates
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-about-updates
+
+toc::[]
+
+You can update a {product-title} cluster by using the OpenShift CLI (`oc`).
+// This PR is for the book build. Note that the OCP structure consists of a landing page of xrefs to other major sections within the book. MicroShift likely does not require that depth of structure, so starting simply with one page.
+
+[id="microshift-about-updates-understanding-microshift-updates"]
+== Understanding {product-title} updates
+{product-title} updates are available as either RPMs or by embedding the {product-title} image in an RPM OSTree blueprint.
+You can update an {product-title} cluster by using the OpenShift CLI (`oc`).
+//Platform administrators can view new update options by looking at the output of the `oc adm upgrade` command.
+//An update begins when...
+
+[NOTE]
+====
+Operators previously installed must be reinstalled using manifests.
+====
+
+=== Using RPM updates
```

Comment:
Do level 3s not get ids? I see some do in the docs, and some don't. I always add them but I couldn't find the rule.

Reply:
All headings are supposed to get anchors.